nucleus/core/kernel/src/main/java/com/sun/enterprise/v3/server/ApplicationLifecycle.java

```diff
@@ -885,7 +928,7 @@ private EngineInfo startEngine(DeploymentContext context, Sniffer sniffer, Strin
         StructuredDeploymentTracing tracing = StructuredDeploymentTracing.load(context);
 
         // start all the containers associated with sniffers.
-        try (DeploymentSpan span = tracing.startSpan(TraceContext.Level.CONTAINER, containerName, DeploymentTracing.AppStage.PREPARE)) {
+        try (DeploymentSpan span = tracing.startSpan(TraceContext.Level.CONTAINER, sniffer.getModuleType(), DeploymentTracing.AppStage.PREPARE)) {
```

Comment:
Shouldn't "Engine" component type be added here?

Reply:
I considered tracking engines separately, but it didn't help the clarity of the result. The engines are tightly coupled with their respective containers, so the distinction doesn't help. (See also this comment above, that mentions sniffers and containers).